src/config.ts

```
@@ -0,0 +1,109 @@
+		viewer: boolean;
+	};
+}
+
+export interface PreviewConfiguration {
+	tableOfContents: boolean;
+	smoothScrolling: boolean;
+	mermaidViewer: boolean;
+}
+
+export interface ConfigurationReader {
+	get<T>(section: string, defaultValue: T): T;
```

ESLint

[error] 'section' is defined but never used.

(no-unused-vars)